index.html

```diff
@@ -29,6 +29,7 @@ <h2 class="section-title">Items</h2>
     <!-- Split Method Section -->
     <section id="split-method-section" class="section">
```

Comment:
I'm just wondering if that elements/classes/ids duplications are needed?
Like `section.section` or `div#items-container.items-container`

Reply:
The class="section" doing something unique in CSS, e.g. `margin-bottom: 20px;` so we should keep it. Similar thing with div#items-container.items-container - both serve different purposes (e.g., the id for JavaScript and the class for styling).

Reply:
Besides this first GH milestone is about getting it all to work properly; I usually do any prettying at the end 😉